### PR TITLE
[K32W] Add commissioning instructions

### PR DESCRIPTION
**Pull request by doru91**
Describe how to setup the topology for NXP E-Lock Demo App.

### CODE REVIEW

**Review by mspang — APPROVED** [review]

File: docs/guides/nxp_k32w_android_commissioning.md [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+2. Connect the smartphone to _OT-BR_ WiFi network;
+3. Open the CHIPTool application on your smartphone;
+4. Tap the _PROVISION CHIP DEVICE WITH THREAD_ button and scan the commissioning
+   QR code. Several notifications will appear, informing you of commissioning
+   progress with scanning, connection, and pairing. At the end of this process,
+   the Thread network settings screen appears.
+
+    ![chiptool_main_screen](../../examples/platform/k32w/doc/images/chiptool_main_screen.png)
+
+5. In the Thread network settings screen, use the default settings and tap the
```
Comment (bzbarsky-apple) [inline docs/guides/nxp_k32w_android_commissioning.md:424]: Again, are these default settings pre-filled or to be copied from the image?

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: docs/guides/nxp_k32w_android_commissioning.md [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+## Forming a Thread network on the Border Router
+
+1.  On the mobile phone connect to the _OT-BR_ Wi-Fi network.
+
+2.  Open a browser and type the IP address 192.168.4.1. The Thread Border Router
+    Web Management page will open.
+
+3.  Navigate to the _Form_ tab then push the _Form_ button using the default
+    parameters:
+    ![nxp_form_nwk](../../examples/platform/k32w/doc/images/form_web.JPG)
```
[the remark this answers is not in the document]
Comment (doru91) [reply docs/guides/nxp_k32w_android_commissioning.md:363]: The default parameters are pre-filled.

File: docs/guides/nxp_k32w_android_commissioning.md (hunk 1 above)
Comment (doru91) [reply docs/guides/nxp_k32w_android_commissioning.md:424]: Pre-filled.

File: examples/lock-app/k32w/README.md [hunk 3]
```diff
@@ -182,20 +182,30 @@ In order to flash the application we recommend using
 [MCUXpresso IDE (version >= 11.0.0)](https://www.nxp.com/design/software/development-software/mcuxpresso-software-and-tools-/mcuxpresso-integrated-development-environment-ide:MCUXpresso-IDE?tab=Design_Tools_Tab).
 
 -   Import the previously downloaded NXP SDK into MCUXpresso IDE. This can be
-    done by drag-and-dropping the SDK archive into MCUXpresso IDE's "Installed
-    SDKs" tab; ![Select SDK](../../platform/k32w/doc/images/select-sdk.JPG)
--   Import Project CHIP in MCUXpresso IDE as Makefile Project:
-    <i>File->Import->C/C++->Existing Code as Makefile Project-> Next </i>.
-    Select Project CHIP folder as the <i>Existing Code Location</i>. In the
```
[the remark this answers is not in the document]
Comment (doru91) [reply examples/lock-app/k32w/README.md:189]: Removed entirely this part and replaced with a link to the official openthread documentation.

File: examples/lock-app/k32w/README.md [hunk 4]
```diff
@@ -182,20 +182,30 @@ In order to flash the application we recommend using
...
-    <i>Toolchain for Indexer Settings </i> list, be sure to keep the setting to
-    <i>none</i>. Click <i>Finish</i>;
--   Configure MCU Settings: right click on project from the workspace and go to
-    <i>Properties->C/C++ Build->MCU Settings</i>. Select K32W061 from the SDK
```
[the remark this answers is not in the document]
Comment (doru91) [reply examples/lock-app/k32w/README.md:193]: Removed entirely this part and replaced with a link to the official openthread documentation.

File: examples/lock-app/k32w/README.md [hunk 5]
```diff
@@ -182,20 +182,30 @@ In order to flash the application we recommend using
...
-    MCUs list;
--   Configure the toolchain editor: <i>C/C++ Build->Tool Chain Editor</i>.
-    Untick the <i>Display compatible toolchains only</i> checkbox. In the
-    drop-down menu named <i>Current toolchain</i>, select <i>NXP MCU Tools</i>.
-    Click <i>Apply and Close</i>;
```
[the remark this answers is not in the document]
Comment (doru91) [reply examples/lock-app/k32w/README.md:198]: Removed entirely this part and replaced with a link to the official openthread documentation.

**Comment by woody-apple** [comment]
@saurabhst ?

**Review by saurabhst — APPROVED** [review]

**Comment by woody-apple** [comment]
@doru91 can you restyle please?

**Review by bzbarsky-apple — APPROVED** [review]

**Review by msandstedt — APPROVED** [review]

**Comment by doru91** [comment]
> @doru91 can you restyle please?

Restyled.

**Pull request merged**